src/df.jl

```diff
@@ -9,18 +9,33 @@ If you want to avoid replacing the symbol, escape it with `^`.
 for strings and symbols respectively.
 """
 macro df(d, x)
-    esc(_df(d,x))
+    syms = Any[]
+    vars = Symbol[]
+    plot_call = _df(d, x, syms, vars)
+    compute_vars = Expr(:(=), Expr(:tuple, vars...), Expr(:call, :(StatPlots.compute_all), d, syms...))
+    esc(Expr(:block, compute_vars, plot_call))
 end
 
-_df(d, x) = x
+_df(d, x, syms, vars) = x
 
-function _df(d, x::Expr)
-    (x.head == :quote) && return :(StatPlots.select_column($d, $x))
+function _df(d, x::Expr, syms, vars)
```

Comment:
Shouldn't this be `_df!` seeing as it modifies the input?

Reply:
Good point!

Reply:
Could the names also be more informative? compute_all seems very broad (should it perhaps be extract_columns_from_table?) and _df a little sparse (parse_table_call or something like that)?

Reply:
Good choice of names, I'll just replace `table` to `iterabletable` to be consistent with the TraitsUtils functions.